neurons/miner.py

```diff
@@ -130,7 +130,7 @@ def __init__(self, config=None):
     def encode(self, examples):
         return self.tokenizer(examples['text'], truncation=True, max_length=512, padding='max_length')
 
-    async def is_alive(self, synapse:  template.protocol.IsAlive) ->  template.protocol.IsAlive:
```

Comment:
@mikkelmedm great work very nice and clean. If we can just refer this to async will help deal with async calls to the miner.